doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex

```diff
@@ -75,9 +75,14 @@ \subsubsection{Predefined Annual Summary Reports}\label{predefined-annual-summar
 \item
   Opaque which includes all opaque surfaces and includes the name of the construction, reflectance, U-Factor, gross area, azimuth, tilt, cardinal direction.
 \item
-  Fenestration which includes all non-opaque surfaces and includes the name of the construction, areas (glass, frame, divider, single opening, multiplied openings), U-Factor, SHGC (the solar heat gain coefficient based on summer conditions), visible transmittance, conductance (frame, divider), indication of shade control, the name of the parent surface, azimuth, tilt, cardinal direction.
+  Fenestration which includes all non-opaque surfaces and includes the name of the construction, areas (glass, frame, divider, single opening, multiplied openings), U-Factor, SHGC (the solar heat gain coefficient based on summer conditions), visible transmittance, NFRC Product Type, assembly U-Factor, assembly SHGC, assembly visible transmittance, conductance (frame, divider), indication of shade control, the name of the parent surface, azimuth, tilt, cardinal direction. 
 \end{itemize}
 
+The assembly result include the effect of the frame and divider and are only produced when WindowProperty:FrameAndDivider input object is used. In addition, the assembly columns are shown for most configurations but are not shown when using Construction:WindowEquivalentLayer and other equivalent layer input object or when using Construction:WindowDataFile with a Window5DataFile.dat file. 
+For exterior fenestration that have a shaded state by using WindowShadingControl the values of the u-factor, SHGC, and visible transmittance are shown for both the glass only and the full assembly in the subtable called Exterior Fenestration Shaded State.
```

Comment:
Is this table at the same level as Exterior Fenestration or a sub-sub-table? If it's at the same level, this should be itemized, leading with Exterior Fenestration Shaded State.